src/components/2021/pie-chart.js

```diff
@@ -15,7 +15,9 @@ const PieChart = (props) => {
       <tr className="cmp-pie-chart__key">
         <th className="cmp-pie-chart__key-name">
           <span className={`cmp-pie-chart__key-marker util-fill-style-${i + 1}`} />
-          {props.dataPoints[i][0]}
+          <span className="cmp-pie-chart__key-text">
```

Comment:
There is not an associated class with this name in the Sass. Did you mean to make a class?

Reply:
Oh yeah, I thought I would need that but didn't.